package_library/packages_windows.go

```diff
@@ -182,6 +169,162 @@ func wuaUpdates(query string) ([]PkgInfo, error) {
 	return updates, nil
 }
 
+// InstallWUAUpdates installs all WUA updates that match query.
+func InstallWUAUpdates(query string) error {
+	connection := &ole.Connection{Object: nil}
+	if err := connection.Initialize(); err != nil {
+		return err
+	}
+	defer connection.Uninitialize()
+
+	unknown, err := oleutil.CreateObject("Microsoft.Update.Session")
+	if err != nil {
+		return err
+	}
+	defer unknown.Release()
+
+	session, err := unknown.QueryInterface(ole.IID_IDispatch)
+	if err != nil {
+		return err
+	}
+	defer session.Release()
+
+	updtsRaw, err := GetWUAUpdateCollection(session, query)
+	if err != nil {
+		return fmt.Errorf("GetWUAUpdates error: %v", err)
+	}
+	defer updtsRaw.Clear()
+
+	updts := updtsRaw.ToIDispatch()
+	count, err := updts.GetProperty("Count")
+	if err != nil {
+		return err
+	}
+	defer count.Clear()
+
+	updtCnt, _ := count.Value().(int32)
```

Comment:
super minor: if this `updtCnt` is the value you care about, it would probably read a little better to call `count` `countRaw` and then `updtCnt` can just be count. Or cleaner still, you can factor this out and call `count,err := getUpdateCount(updts)`